.travis.yml

```diff
@@ -26,8 +26,12 @@ install:
   - if [[ $TRAVIS_PYTHON_VERSION == '3.3' ]]; then conda create --yes -q -n pyenv mkl python=3.3 numpy scipy nose pyparsing pip flake8 six pep8 pyflakes; fi
   - if [[ $TRAVIS_PYTHON_VERSION == '3.4' ]]; then conda create --yes -q -n pyenv mkl python=3.4 numpy scipy nose pyparsing pip flake8 six pep8 pyflakes; fi
   - source activate pyenv
+  - curl -O https://raw.githubusercontent.com/ASalvail/blocks/master/req-travis-conda.txt
+  - conda install -q --yes python=$TRAVIS_PYTHON_VERSION mkl --file req-travis-conda.txt
   - pip install theano
   - pip install https://github.com/SMART-Lab/smartlearner/archive/master.zip
+  - pip install -q -r https://raw.githubusercontent.com/ASalvail/blocks/master/req-travis-pip.txt
+  - pip install -q git+git://github.com/ASalvail/blocks.git -r https://raw.githubusercontent.com/ASalvail/blocks/master/requirements.txt
```

Comment:
I can't, since this is using the PR I made for blocks. It'd fail otherwise. 

On August 14, 2015 11:50:50 PM EDT, "Marc-Alexandre Côté" <notifications@github.com> wrote:
>>    - pip install theano
>>    - pip install
>https://github.com/SMART-Lab/smartlearner/archive/master.zip
>> +  - pip install -q -r
>https://raw.githubusercontent.com/ASalvail/blocks/master/req-travis-pip.txt
>> +  - pip install -q git+git://github.com/ASalvail/blocks.git -r
>https://raw.githubusercontent.com/ASalvail/blocks/master/requirements.txt
>
>```
>- pip install -q git+git://github.com/mila-udem/blocks.git -r
>https://raw.githubusercontent.com/mila-udem/blocks/master/requirements.txt
>```
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/SMART-Lab/smartmodels/pull/15/files#r37132896

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.